app/v4/retrieveChargeHistoryByChargeReference/RetrieveChargeHistoryByChargeReferenceConnector.scala

```diff
@@ -38,12 +38,40 @@ class RetrieveChargeHistoryByChargeReferenceConnector @Inject() (val http: HttpC
       correlationId: String): Future[DownstreamOutcome[RetrieveChargeHistoryResponse]] = {
 
     import request._
-    import schema._
 
-    val queryParams = List("chargeReference" -> chargeReference.value)
+    val IfsQueryParams = List("chargeReference" -> chargeReference.value)
+
+    val additionalContractHeaders: Seq[(String, String)] = List(
+      "X-Message-Type"        -> "ETMPGetChargeHistory",
+      "X-Originating-System"  -> "MDTP",
+      "X-Receipt-Type"        -> "ITSA",
+      "X-Receipt-Date"        -> isoDateTimeStamp,
+      "X-Transmitting-System" -> "HIP"
+    )
```

Comment:
make this val start lowercase `ifsQueryParams`

Change the `X-Receipt-Type` header to `X-Regime-Type`

Please make these changes across the versions and both endpoints